### PR TITLE
Fix ancestors example

### PR DESCRIPTION
**Pull request by vsbogd**

### CODE REVIEW

**Comment by vsbogd** [comment]
@ngeiswei could you please take a look, it is almost trivial, on the other hand I would like to be sure it aligns with your understanding.

**Comment by ngeiswei** [comment]
@vsbogd, yes it aligns, however as I'm reviewing your PR I realize that I've messed up some patterns. For instance `deduction-rule` is supposed to have specializations like `inheritance-deduction-rule`. The only reason it doesn't break is due to backward compatibility. Others are blatantly incorrect for instance `^.*condition-negation-.+-rule$` cannot match the current condition negation rules (as they also couldn't before your PR).

I'm gonna merge for now, I think the use of delimiters `^` and `$` is a good idea. Then I'll check and fix the breaking patterns shortly.

File: examples/pln/ancestors/ancestors.scm [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ;; Load PLN (with only the rule that matters)
 (use-modules (opencog pln))
 (pln-load 'empty)
```
Comment (ngeiswei) [inline examples/pln/ancestors/ancestors.scm:3]: BTW, this is no logger necessary (unless ones want to load the whole 'standard rule-base, which is probably a bad idea in general).

Comment (vsbogd) [reply examples/pln/ancestors/ancestors.scm:3]: Thank you, I though about it but was not confident.

**Pull request merged**